apps/mull-api/src/database/factories/event.factory.ts

```diff
@@ -0,0 +1,20 @@
+import * as Faker from 'faker';
+import { define, factory } from 'typeorm-seeding';
+import { Event, Media, User } from '../../app/entities';
+
+define(Event, (faker: typeof Faker) => {
+  const event = new Event();
+
+  event.title = faker.lorem.lines();
+  event.description = faker.lorem.paragraph();
+  event.startDate = faker.date.future();
+  event.endDate = faker.date.future();
```

Comment:
This won't guarantee that the date will be valid (start might be before date). Use soon

Example: https://github.com/Marak/faker.js/blob/master/test/date.unit.js#L83


```suggestion
  event.startDate = faker.date.soon(5);
  event.endDate = faker.date.soon(6);
```

Reply:
I believe faker.data.soon is deprecated and is no longer a function

Reply:
I don't see any mention of that in their docs and it is definitely a function: https://github.com/Marak/faker.js/blob/master/lib/date.js#L105

You're getting that error because you're using an older version of faker, which is being used by typeorm-seeding

I changed my file to the following and the sending worked fine. Id recommend this as a fix:
```ts
define(Event, () => {
  const event = new Event();

  event.title = faker.lorem.lines();
  event.description = faker.lorem.paragraph();
  event.startDate = faker.date.soon(3);
  event.endDate = faker.date.soon(5);
  event.restriction = faker.random.number(3);

  event.host = factory(User)() as any;
  event.coHosts = factory(User)().createMany(faker.random.number(5)) as any;
  event.participants = factory(User)().createMany(faker.random.number(30)) as any;
  event.image = factory(Media)() as any;

  return event;
});
```

Reply:
faker.date.soon(number) gives :
![Screenshot 2021-01-19 151606](https://user-images.githubusercontent.com/44760251/105087976-4889fa00-5a69-11eb-9da0-0454528d8213.png)